src/main/java/com/theironyard/services/StripeService.java

```diff
@@ -0,0 +1,52 @@
+package com.theironyard.services;
+
+import com.stripe.Stripe;
+import com.stripe.exception.*;
+import com.stripe.model.Charge;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class StripeService {
+
+    @Value("${stripe.key.secret}")
+    private String key;
+
+    @PostConstruct
+    public void init(){
+        Stripe.apiKey = key;
+    }
+
+    /* *
+     * Takes a "token" generated by Strip Api and an "amount",
+     * creates a charge object and submits it to Stripe for processing
+     * */
+    public String chargeCard(String token, int amount){
+        Map<String, Object> params = new HashMap<>();
+        params.put("amount", amount);
+        params.put("currency", "usd");
+        params.put("description", "Monthly Subscription to Text Alarm");
+        params.put("source", token);
+
+        String id = null;
+        try {
+            Charge charge = Charge.create(params);
+            id = charge.getId();
+        } catch (AuthenticationException e) {
+            e.printStackTrace();
```

Comment:
In the future you will want to do something with these error message to communicate back to the user what happened.  Or even log out to your log files.